UnrealPlugin/Source/PlayFabGSDK/Public/GSDKUtils.h

```diff
@@ -103,10 +105,21 @@ class PLAYFABGSDK_API UGSDKUtils : public UBlueprintFunctionLibrary
 	UFUNCTION(BlueprintCallable, Category="PlayFab|GSDK|Callbacks")
 	static void RegisterGSDKHealthCheckDelegate(const FOnGSDKHealthCheck_Dyn& OnGSDKHealthCheckDelegate);
 
-	// Register the GSDK Maintenance Delegate, which gets called if the VM this server is running on expects to go into Maintenance soon
-	UFUNCTION(BlueprintCallable, Category="PlayFab|GSDK|Callbacks")
+	/// <summary>
+	/// DEPRECATED Use RegisterGSDKMaintenanceV2Delegate. Register the GSDK Maintenance Delegate, which gets called if the VM this server is running on expects to go into Maintenance soon.
```

Comment:
thinking about this, do we want to mark the old API as deprecated? This implies that we'll eventually remove it, which I don't think we'll ever do.

Reply:
We don't have to necessarily remove it, but with V2 in, there's no reason to use the old one at all. Though we can have a more in-depth discussion about it in the group chat to get everyone's input on how to handle this if you want 👍